### PR TITLE
Rm maintainer + migrate

### PR DESCRIPTION
**Pull request by muellerzr**
# What does this PR do?

Adjusts the relevant areas of the PR template and issue template now that Sourab has left. 

TL;DR:

* I will be taking over the FSDP and DeepSpeed issues, and eventually more maintainers will/are getting training that can assist as well.
* @SunMarc will be helping out with the core Trainer areas (Thanks Marc!) 🤗 

Fixes # (issue)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#create-a-pull-request),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

@amyeroberts @LysandreJik

### CODE REVIEW

**Review by amyeroberts — APPROVED** [review]
Thanks for updating! 

Bit of a sad update 😢

**Review by ArthurZucker — APPROVED** [review]
